src/site/site.xml

```diff
@@ -84,19 +84,19 @@
       <item name="Release Notes" href="https://issues.apache.org/jira/browse/HBASE?report=com.atlassian.jira.plugin.system.project:changelog-panel#selectedTab=com.atlassian.jira.plugin.system.project%3Achangelog-panel" />
       <item name="Code Of Conduct" href="coc.html"/>
       <item name="Blog" href="http://blogs.apache.org/hbase/"/>
-      <item name="Mailing Lists" href="mailing-lists.html"/>
-      <item name="Team" href="team.html"/>
+      <item name="Mailing Lists" href="mail-lists.html"/>
+      <item name="Team" href="team-list.html"/>
       <item name="ReviewBoard" href="https://reviews.apache.org/"/>
       <item name="HBase Sponsors" href="sponsors.html"/>
       <item name="Thanks" href="https://www.apache.org/foundation/thanks.html"/>
       <item name="Powered by HBase" href="poweredbyhbase.html"/>
       <item name="Other resources" href="resources.html"/>
     </menu>
     <menu name="Project Information">
-      <item name="Project Summary" href="summary.html"/>
+      <item name="Project Summary" href="project-summary.html"/>
       <item name="Dependency Information" href="dependency-info.html"/>
-      <item name="Source Repository" href="scm.html"/>
-      <item name="Issue Tracking" href="issue-management.html"/>
+      <item name="Source Repository" href="source-repository.html"/>
+      <item name="Issue Tracking" href="issue-tracking.html"/>
```

Comment:
Why these changes? The site works with them in place?

Reply:
These files are generated by Maven Project Info Report tool and from 3.0 the filenames are changed. With the rollback to version 2.9 the filenames need to be changed back as well.